### PR TITLE
filebeat/input/journald: fix field name translation

### PR DESCRIPTION
**Pull request by belimawr**
## What does this PR do?

The field names from journald were not being translated to our format
when sending the event to the output. This commit fixes it.

## Why is it important?

It fixes the Journald event format

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- ~~[ ] I have made corresponding changes to the documentation~~
- ~~[ ] I have made corresponding change to the default configuration files~~
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

~~## Author's Checklist~~

## How to test this PR locally

Run Filebeat with the journald input and check that the event fields are translated as stated on our [documentation](https://www.elastic.co/guide/en/beats/filebeat/8.0/filebeat-input-journald.html#filebeat-input-journald-translated-fields)

From the `filebeat` folder, you can use this yaml as configuration. It points to a journal used on unit tests with a handful of events
```yaml
filebeat.inputs:
- type: journald
  enabled: true
  paths: ["input/journald/testdata/input-multiline-parser.journal"]
  save_remote_hostname: true

output.console:
  codec.json:
    pretty: true
  
processors:
  - add_host_metadata:
      when.not.contains.tags: forwarded
```

## Related issues

- Fixes https://github.com/elastic/beats/issues/30031

~~## Use cases~~
~~## Screenshots~~
~~## Logs~~

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -98,6 +98,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Filebeat*
 
 - Fix using log_group_name_prefix in aws-cloudwatch input. {pull}29695[29695]
+- Fix journald event format
```
Comment (andrewkroh) [inline CHANGELOG.next.asciidoc:101]: Has this bug ever been included in a release? If we caught it before it was released then omit the changelog entry.

File: filebeat/input/journald/input.go [hunk 2]
```diff
@@ -251,9 +261,17 @@ func (r *readerAdapter) Next() (reader.Message, error) {
 	content := []byte(data.Fields["MESSAGE"])
 	delete(data.Fields, "MESSAGE")
 
-	fields := make(map[string]interface{}, len(data.Fields))
-	for k, v := range data.Fields {
-		fields[k] = v
+	fields := r.converter.Convert(data.Fields)
+	fields.Put("event.kind", "event")
+	fields.Put("event.created", data.RealtimeTimestamp)
```
Comment (andrewkroh) [inline filebeat/input/journald/input.go:266]: [`event.created`](https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-created) is the time Filebeat generated this document/event (usually this means`time.Now().UTC()`). So I think this is incorrect (w/o knowing for sure what RealtimeTimestamp is).

`@timestamp` should the time from the journald event.

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (belimawr) [reply CHANGELOG.next.asciidoc:101]: No it haven't, I'll remove it

File: filebeat/input/journald/input.go (hunk 2 above)
Comment (belimawr) [reply filebeat/input/journald/input.go:266]: Good catch!

**Comment by belimawr** [comment]
@andrewkroh I addressed your comments on 2e7a8e9c80

**Review by kvch — APPROVED** [review]
WFG

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**